src/Microsoft.DotNet.ProjectJsonMigration/project.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "warningsAsErrors": true
+  },
+  "dependencies": {
+    "Microsoft.DotNet.Compiler.Common": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.Cli.Utils": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "Microsoft.Win32.Registry": {
+      "version": "4.0.0",
+      "exclude": "compile"
+    },
+    "MSBuild": "0.1.0-preview-00029-160805",
```

Comment:
The only ones of these that I think are necessary are:

```
    "Microsoft.Build.Framework"
    "Microsoft.Build.Utilities.Core"
    "Microsoft.Build"
```

---
In reply to: [75894907](https://github.com/dotnet/cli/pull/4074#discussion_r75894907) [](ancestors = 75894907)